cypress/e2e/wavehand-spec.cy.ts

```diff
@@ -0,0 +1,41 @@
+describe("WaveHandPicker in spec", () => {
+  beforeEach(() => {
+    cy.visit("/");
+  });
+
+  it("should allow selecting, adding, and deleting wave hand buttons", () => {
+    // select the first button
+    cy.get(".wave-hand-button").first().as("button");
```

Comment:
Same comment as the unit tests about how the tests should ideally be testing stuff the users can see, rather than classes that are only part of the code.